.github/workflows/cleaner.yml

```diff
@@ -0,0 +1,53 @@
+name: Cleaner
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+      - name: Install Dependencies
+        run: npm install
+      - name: Cleanup
+        run: node .github/workflows/cleaner.js
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Config
+        run: |
+          git config --global user.name 'Hexo'
+          git config --global user.email 'hexojs@users.noreply.github.com'
```

Comment:
I don't know who can control over this [account](https://github.com/hexo). If it publishes awful content in its own page, it may affect our credibility

Reply:
This account seems to have not been used for a long time, I hope it doesn't cause any confusion.